### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by polarstormx**
Added tutorials for running examples on different machines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,9 +50,31 @@ mkdir testmedia
...
+            "dest_ip": "recever_ip",
+            "dest_port": "8888"
+        },
+```
+**Make sure your sender machine can access receiver machine's 8888 TCP port and any UDP ports.**
+
+Then run the following command, and check if the `outvideo.yuv` and `outaudio.wav` are generated at the receiver machine.
+```bash
+# receiver
+docker run -d --net==host --rm -v  `pwd`:/app -w /app --name alphartc_pyinfer opennetlab.azurecr.io/challenge-env peerconnection_serverless receiver_pyinfer.json
```
Comment (jeongyooneo) [inline README.md:76]: Maybe `--network=host` instead of `--net==host` ?

**Comment by jeongyooneo** [comment]
Thank you for your PR, @polarstormx ! 
I'm currently trying the submission verification locally, and I think adding `--network=host` is also needed for local run.

Running the below locally leaves `(network.cc:885): Connect failed with 99` error in `webrtc.log`:

```
docker run -d --rm -v `pwd`:/app -w /app --name alphartc_pyinfer opennetlab.azurecr.io/challenge-env peerconnection_serverless receiver_pyinfer.json
docker exec alphartc_pyinfer peerconnection_serverless sender_pyinfer.json
```

Adding `--network=host` when running the receiver eliminates this error:

```
docker run -d --network=host --rm -v  `pwd`:/app -w /app --name alphartc_pyinfer opennetlab.azurecr.io/challenge-env peerconnection_serverless receiver_pyinfer.json
```

**Comment by polarstormx** [comment]
I have modify that part you mentioned, but Azure pipelines prompts me "ALL checks have failed". What should I do to fix it?

**Comment by jeongyooneo** [comment]
> I have modify that part you mentioned, but Azure pipelines prompts me "ALL checks have failed". What should I do to fix it? @[jeongyooneo](https://github.com/jeongyooneo)

The script used for CI test ([azure-pipelines.yml](https://github.com/OpenNetLab/Challenge-Example/blob/master/azure-pipelines.yml)) needs numpy. I made a quick fix in [a separate PR](https://github.com/OpenNetLab/Challenge-Example/pull/10). You can reference this PR and fix your azure-pipelines.yml accordingly :-)

**Pull request merged**